ring/mod.go

```diff
@@ -26,6 +26,9 @@ func makeMod(r *models.Ring) (torus.Ring, error) {
 		rep = 1
 	}
 	pil := torus.PeerInfoList(r.Peers)
+	if rep > len(pil) {
+		clog.Noticef("Requested replication level %d, but has only %d peers. Add nodes to match replication.", rep, len(pil))
```

Comment:
If we return error here, ring will not be registered until enough number of nodes are running. It means that `torusctl ring manual-change --type=mod` needs to run with running storage nodes.
Also, it would be better to make same manner with ring ketama.